docs/UserGuide.md

```diff
@@ -1,4 +1,4 @@
-# AddressBook Level 4 - User Guide
+# Main - User Guide
```

Comment:
Is the name of your product "Main"? If not, better to include the name here than `Main`.